src/modules/outliers/xyStats.js

```diff
@@ -0,0 +1,32 @@
+// most performant way of getting min and max
+export const getXYStats = points => {
+    let xSum = 0
+    let ySum = 0
+    let xMin = points[0][0]
+    let xMax = xMin
+    let yMin = points[0][1]
+    let yMax = yMin
+    let i
+    let x
+    let y
+
+    for (i = 1; i < points.length; i++) {
+        x = points[i][0]
+        y = points[i][1]
+        xSum += xMin
```

Comment:
What I meant was actually to initialise `xSum` and `ySum` outside the for loop, using `xMin` and `yMin`.
Here it should use `x` and `y`.

Reply:
Yes, fixing this now together with the test